### PR TITLE
feat(STONEINTG-892): update doc for override snapshot

### PR DESCRIPTION
**Pull request by hongweiliu17**
* update doc to add snapshotComponent's source for override snapshot

Signed-off-by: Hongwei Liu <hongliu@redhat.com>

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/testing/integration/creatingoverridesnapshot.adoc [hunk 1]
```diff
@@ -30,19 +30,22 @@ spec:
...
       containerImage: quay.io/redhat-user-workloads/ws-sample-tenant/application-sample/component-sample@sha256:0db0a473a6abf5c15c424ab07cfbd5c40c06622fe648d4fe6a6b6abc224a0d0c <3>
+      source: <4>
+        git:
+          revision: fa8b89274a61ef0f1c257b7a84c37aa2ec844109
 ----
 <1> The label `test.appstudio.openshift.io/type: override` that indicates this is an `override` snapshot.
 <2> The component name you will reset its Global Candidate List.
 <3> The container image with valid digest you will reset the component <2> to.
-
+<4> Optional. The component source.
```
Comment (dirgim) [inline docs/modules/ROOT/pages/how-tos/testing/integration/creatingoverridesnapshot.adoc:40]: Maybe we can expand this to say `Optional. The component source containing the git revision.`

Comment (hongweiliu17) [reply docs/modules/ROOT/pages/how-tos/testing/integration/creatingoverridesnapshot.adoc]: Updated. Thanks

**Review by dirgim — APPROVED** [review]
LGTM

**Review by kasemAlem — APPROVED** [review]

**Comment by hongweiliu17** [comment]
@ralphbean Could you help to review and merge this PR? Thanks!

**Comment by simonbaird** [comment]
FYI I created a little update in the appstudio docs for this, see https://github.com/redhat-appstudio/docs.appstudio.io/pull/276/files .

You could include it here, or merge this and include it another PR.

**Comment by hongweiliu17** [comment]
> FYI I created a little update in the appstudio docs for this, see https://github.com/redhat-appstudio/docs.appstudio.io/pull/276/files .
> 
> You could include it here, or merge this and include it another PR.

Thanks! PR is updated.

**Review by dirgim — APPROVED** [review]
LGTM after updates

**Review by simonbaird — APPROVED** [review]

**Pull request merged**